source/server/server.cc

```diff
@@ -62,18 +63,14 @@ InstanceImpl::InstanceImpl(Options& options, Network::Address::InstanceConstShar
       }
     }
 
-    failHealthcheck(false);
-
     uint64_t version_int;
```

Comment:
nit: instead of passing version_int, can you just do this where we now set the stat?